.xrefcheck.yaml

```diff
@@ -2,56 +2,27 @@
 #
 # SPDX-License-Identifier: Unlicense
```

Comment:
We're using two `xrefcheck: ignore link` annotations in `docs/building.md` and in `baseDAO/haskell/test/README.md`, I think we can delete both.

Reply:
done